lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -166,12 +166,12 @@ def setup(self):
                 )
 
         filename = self.output_irf_file.name
-        if filename.split(".")[1] != "fits":
+        if filename.split(".")[-2] != "fits" or filename.split(".")[-1] != "gz":
             self.log.warning(
                 f"{filename} is not a correct "
-                "compressed FITS file name. It will be corrected."
+                "compressed FITS file name (.fits.gz). It will be corrected."
             )
-            filename = filename.split(".")[0] + ".fits.gz"
+            filename += ".fits.gz"
```

Comment:
Won't this just add to a wrong name, like `irf.xyz` -> `irf.xyz.fits.gz`, instead of just taking care of the extensions?